src/pages/tBTC/Bridge/Minting/MakeDeposit.tsx

```diff
@@ -41,77 +52,61 @@ const AddressRow: FC<
-  return (
-    <Card {...restProps} backgroundColor={bgColor} border="none">
-      {children}
-    </Card>
-  )
-}
-
 const BTCAddressSection: FC<{ btcDepositAddress: string }> = ({
   btcDepositAddress,
 }) => {
-  const titleColor = useColorModeValue("gray.700", "gray.100")
-  const btcAddressColor = useColorModeValue("brand.500", "white")
+  const { onCopy: handleCopy } = useClipboard(btcDepositAddress)
 
   return (
-    <>
+    <VStack spacing="2" align="start">
       <HStack
         alignItems="center"
-        mb="3.5"
         // To center the tooltip icon. The tooltip icon is wrapped by `span`
         // because of: If you're wrapping an icon from `react-icons`, you need
         // to also wrap the icon in a `span` element as `react-icons` icons do
         // not use forwardRef. See
         // https://chakra-ui.com/docs/components/tooltip#with-an-icon.
         sx={{ ">span": { display: "flex" } }}
       >
-        <BodyMd color={titleColor}>BTC Deposit Address</BodyMd>
-        <TooltipIcon
-          color={titleColor}
-          label="This is a unique BTC address generated based on the ETH address and Recovery address you provided. Send your BTC funds to this address in order to mint tBTC."
-        />
+        <BodySm fontWeight="medium" color="brand.100" lineHeight={1.5}>
+          BTC Deposit Address
+        </BodySm>
+        <TooltipIcon label="This is a unique BTC address generated based on the ETH address and Recovery address you provided. Send your BTC funds to this address in order to mint tBTC." />
       </HStack>
-      <BTCAddressCard p="2.5" display="flex" justifyContent="center">
-        <Box
-          p={2.5}
-          backgroundColor={"white"}
-          width={"100%"}
-          maxW="205px"
-          borderRadius="8px"
-        >
+      <HStack
+        align="start"
+        py={4}
+        pl={4}
+        pr={10}
+        spacing={10}
+        rounded="2xl"
+        border="1px solid #333"
+      >
+        <Box p={3.5} rounded="xl" bg="brand.100">
           <QRCode
-            size={256}
-            style={{ height: "auto", maxWidth: "100%", width: "100%" }}
+            size={180}
             value={btcDepositAddress}
-            viewBox={`0 0 256 256`}
+            viewBox="0 0 180 180"
+            bgColor="transparent"
           />
         </Box>
-      </BTCAddressCard>
-      <CopyToClipboard textToCopy={btcDepositAddress}>
-        <HStack mt="2.5">
-          <BTCAddressCard minW="0" p="2">
-            <BodyMd color={btcAddressColor} textStyle="chain-identifier">
-              {btcDepositAddress}
-            </BodyMd>
-          </BTCAddressCard>
-          <BTCAddressCard
-            flex="1"
-            p="4"
-            display="flex"
-            alignSelf="stretch"
-            alignItems="center"
+        <VStack spacing={6} align="start">
+          <BodyLg color="brand.100" overflowWrap="anywhere">
+            {btcDepositAddress}
```

Comment:
Non- blocking:
I think that on some resolutions we could benefit if the address was placed below the qr code instead next to it:

WIdth: 1023
<img width="463" alt="image" src="https://github.com/threshold-network/bitcoin-on-base/assets/40306834/a0ce815b-b9bb-47aa-9513-2ec83e8651b8">